adapters/handlers/rest/clusterapi/serve.go

```diff
@@ -77,3 +90,20 @@ func index() http.Handler {
 		json.NewEncoder(w).Encode(payload)
 	})
 }
+
+// staticRoute is used to convert routes in our internal http server into static routes
+// by removing all the dynamic variables in the route. Useful for instrumentation
+// where "route cardinality" matters.
+
+// Example: `/replicas/indices/Movies/shards/hello0/objects` -> `/replicas/indices`
```

Comment:
hm, are we not interested in the `/shards/{shardID}/objects` part at all? I guess with the current setup it is very hard to parse these details in a uniform way :/

Reply:
Parsing the detail is one problem sure. Other goal here is to have the same pattern we add to the mux router. If you look at that file, we add `/replicas/indices` in the mux not `/replicas/indices/{class}/shards/{shardID}/objects` for example.

Easy to find via simple "grep" for the route -> handler in the code base.

Reply:
> Yeah, but if there are so much functionality packed under one namespace (e.g. under /replicas/indices there are ~10 different handlers) then it might be very hard to pinpoint which endpoint is actually causing troubles.

> (e.g. under /replicas/indices there are ~10 different handlers) 

Each of this `replicas/indices` has single http handler. That's why used mux to match this pattern.

> BTW, we are including HTTP method in the metric, right?

Yes.

Reply:
ok, let's try for now, we can extend this later